src/clients/BundleDataClient.ts

```diff
@@ -194,13 +199,27 @@ export class BundleDataClient {
       }, {} as Record<number, Record<string, UnderlyingType>>);
 
     const data = persistedData[0].data;
-    return {
+    const bundleData = {
       bundleFillsV3: convertTypedStringRecordIntoNumericRecord(data.bundleFillsV3),
       expiredDepositsToRefundV3: convertTypedStringRecordIntoNumericRecord(data.expiredDepositsToRefundV3),
       bundleDepositsV3: convertTypedStringRecordIntoNumericRecord(data.bundleDepositsV3),
       unexecutableSlowFills: convertTypedStringRecordIntoNumericRecord(data.unexecutableSlowFills),
       bundleSlowFillsV3: convertTypedStringRecordIntoNumericRecord(data.bundleSlowFillsV3),
     };
+    this.logger.debug({
+      at: "BundleDataClient#loadPersistedDataFromArweave",
+      message: `Loaded persisted data from Arweave in ${Math.floor(Date.now() - start) / 1000}s.`,
```

Comment:
The rounding function should include the division. nb. we actually have an `msToS()` helper in src/utils.

```suggestion
      message: `Loaded persisted data from Arweave in ${Math.round(performance.now() - start / 1000)}s.`,
```